### ISSUE TITLE
Code comments missing?

### ISSUE THREAD

**Issue by dpriskorn** [internetarchive/newsum#1]
I tried reading the code and was missing comments about reasons for building it this way
E.g. why use this model from gpt 3? Why use these parameters?
What is streamlit?
Why use streamlit?

**Comment by ibnesayeed** [comment]
I agree, the code is not documented, as it is something I put together while exploring things and experimenting with options. It sure needs some polish.

**Comment by Kriti382** [comment]
Yup! A piece is missing...I would like to fix this issue.
